cmd/all.go

```diff
@@ -17,7 +17,7 @@ var auditAllCmd = &cobra.Command{
 
 Example usage:
 kubeaudit all -f /path/to/yaml`,
-	Run: runAudit(mergeAuditFunctions(allAuditFunctions)),
+	Run: runAudit(mergeAuditFunctions(allAuditFunctions), getResources),
```

Comment:
I made a few changes in such a way that we only need the `getResources()`. My initial main motivation was to reduce the overhead when running audit checks (assume we run all these check on really large cluster with many objects). Actually I think this would be a nice thing for an additional refactoring.